docs/ad/settings.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+title: Settings
+parent: Anomaly Detection
+nav_order: 4
+---
+
+# Settings
+
+The anomaly detection plugin adds several settings to the standard Elasticsearch cluster settings.
+They are dynamic, so you can change the default behavior of the plugin without restarting your cluster.
+You can mark them `persistent` or `transient`.
+
+To update the retention period of the result index:
```

Comment:
Should we have "for example" indicating this is one example of changing setting, but you can do the same thing for other settings in the following table?